meas/MeasUDF/test/azel.cpp

```diff
@@ -0,0 +1,39 @@
+//# azel.cc: utility to print azimut and elevation of a given source
+
+#include <casacore/tables/TaQL/TableParse.h>
+
+using namespace casacore;
+using namespace std;
+
+int main(int argc, char** argv) {
+  string source("Jupiter"), observatory("DWL"), time("datetime()");
+
+  if (argc==1) {
+    cout << "Usage: azel [source] [observatory] [time]" << endl;
+    cout << "  source: default Jupiter" << endl;
+    cout << "  observatory: default DWL" << endl;
+    cout << "  time: default now, example (UTC) \"2016-06-28 19:54\"" << endl;
+    exit(0);
+  }
+  if (argc > 1) {
+    source = argv[1];
+  }
+  if (argc > 2) {
+    observatory = argv[2];
+  }
+  if (argc > 3) {
+    time = argv[3];
+  }
+
+  string query = "calc meas.azel(";
+  query += "'" + source + "', ";
+  query += time + ", ";
+  query += "'" + observatory + "'";
+  query += ") deg";
```

Comment:
Maybe nice to have another argument to choose the output units

Reply:
Will do this in a future version. Perhaps. Maybe.